ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml

```diff
@@ -0,0 +1,27 @@
+build_root:
+  image_stream_tag:
+    namespace: coreos
+    name: cosa-buildroot
+    tag: latest
+
+# With this we build a container image derived from cosa-buildroot
+# that contains new rpm-ostree binaries.  From there...since this
+# cluster doesn't support recursive containerization or virt,
+# we can't do much interesting to test it today.  But hopefully
+# that will soon change.
+# https://github.com/openshift/release/issues/4892
+binary_build_commands: env SKIP_INSTALLDEPS=1 ./ci/build-check.sh
+
+tests:
+- as: sanity
+  commands: rpm-ostree --version
+  container:
+    from: bin
+
+resources:
+  '*':
+    limits:
+      memory: 6Gi
```

Comment:
That's a lot of memory for your sanity test :)

Reply:
Hm yeah, the build process (gcc and Rust compilers) is what uses RAM; could dial that back down some.

Reply:
Ultimately I think what we really want is to

1) Attribute resource usage to repositories and organizations and keep it sane
2) https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler